### ISSUE TITLE
Basic Install Doesn't Seem to Work

### ISSUE THREAD

**Issue by chaseconey** [ansible-semaphore/semaphore#6]
Just ran through the quickstart installation and it doesn't appear to be working...

Version information:
**OS:** OSX 10.10.2 (newest)
**vagrant:** 1.7.2

`vagrant up` runs perfectly fine, all steps are run with no issues but http://localhost:3000 doesn't serve up anything. What are some steps to figure out what is going on here?

Thanks

**Comment by leopoul** [comment]
Run into the same issue. From the latest commit's description "Fix authentication failures & get rid of grunt" it seems that grunt is been taken off. So, I guess this no longer works https://github.com/ansible-semaphore/semaphore/blob/master/playbooks/run_semaphore#L4 and it should go via node.
After failing I went with vagrant ssh and flowed the steps of README:

1. `cp lib/credentials.example.json lib/credentials.json` (<- make custom to your environment)
2. `bower install`
3. `node bin/semaphore`

(2) is probably not needed.

However now I get a:
```
GET /vendor/requirejs/require.js 200 15.882 ms - 764
  LESS File error : '../vendor/fontawesome/less/font-awesome.less' wasn't found. Tried - /opt/semaphore/public/vendor/fontawesome/less/font-awesome.less,../vendor/fontawesome/less/font-awesome.less
  LESS File : /opt/semaphore/public/css/semaphore.less 13:0
Error: '../vendor/fontawesome/less/font-awesome.less' wasn't found. Tried - /opt/semaphore/public/vendor/fontawesome/less/font-awesome.less,../vendor/fontawesome/less/font-awesome.less
```
whenever I hit localhost:3000 from my host

**Comment by craftbyte** [comment]
It seems like the issue is that the vendor folder doesn’t exist. Seems like the dependencies are not in there.

**Comment by leopoul** [comment]
Got it working eventually.
Step 2 is needed `bower install` 
However, running the above I got:
`bower ECMDERR       Failed to execute "git ls-remote --tags --heads git://github.com/FortAwesome/Font-Awesome.git", exit code of #128`

Googling around, I fixed the prob with:
```
git config --global url."https://".insteadOf git://
bower cache clean
bower install --allow-root
```

My semaphore is up. It definitely needs some looking into. If I find time I'll try to send a patch
